components/molecule/pagination/src/index.js

```diff
@@ -200,7 +230,20 @@ MoleculePagination.defaultProps = {
   selectedPageButtonDesign: 'solid',
   nonSelectedPageButtonDesign: 'flat',
   prevButtonDesign: 'flat',
-  nextButtonDesign: 'flat'
+  nextButtonDesign: 'flat',
+  createUrl: ({pageNumber, urlPattern}) => {
```

Comment:
Could you decouple this default functions? Also, could you use default parameters?

Reply:
Yes, better move these `defaultProps` to default params in the component function.